tests/services/test_scheduler.py

```
@@ -97,24 +98,199 @@ def test_decrypt_location_job(self, db_session, storage_location, file_inventory
+        monkeypatch.setattr("app.services.scheduler.SchedulerSessionLocal", MagicMock(return_value=db_session))
+
+    def _make_cold_location(self, db_session, path: str, name: str = "Cold Storage") -> ColdStorageLocation:
+        loc = ColdStorageLocation(name=name, path=path)
+        db_session.add(loc)
+        db_session.commit()
+        db_session.refresh(loc)
+        return loc
+
+    def _make_hot_path(self, db_session, path: str, name: str = "Hot Path") -> MonitoredPath:
+        from app.models import ColdStorageLocation, MonitoredPath
+        cold = ColdStorageLocation(name=f"Cold for {name}", path="/tmp/cold_dummy")
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/cold_dummy"

(S108)